sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceStrategySuite.scala

```diff
@@ -529,37 +529,39 @@ class FileSourceStrategySuite extends QueryTest with SharedSparkSession with Pre
   }
 
   test("SPARK-32019: Add spark.sql.files.minPartitionNum config") {
-    withSQLConf(SQLConf.FILES_MIN_PARTITION_NUM.key -> "1") {
-      val table =
-        createTable(files = Seq(
-          "file1" -> 1,
-          "file2" -> 1,
-          "file3" -> 1
-        ))
-      assert(table.rdd.partitions.length == 1)
-    }
+    withSQLConf(SQLConf.FILES_MAX_PARTITION_BYTES.key -> "2MB") {
```

Comment:
Can we add the config `spark.sql.files.openCostInBytes` ? The result is based on it although we don't change the default value.

Reply:
Done